### ISSUE TITLE
Add AppImage and Flatpak binaries for 1.1.0rc3

### ISSUE THREAD

**Issue by l0f4r0** [GourmandRecipeManager/gourmand#124]
There is no AppImage or Flatpak binaries for Gourmand 1.1.0rc3 currently (only source code).

Could they be added to the [GitHub release page](https://github.com/GourmandRecipeManager/gourmand/releases/tag/1.1.0rc3) please?

**Comment by cydanil** [comment]
Yes, will do! There's been issues building them, which I'm in the process of fixing up.

**Comment by Oldbwl** [comment]
Awaiting these myself. I can only test from the AppImage so cannot yet provide any feedback

**Comment by FriedrichFroebel** [comment]
It should re rather easy to at least build the AppImage for every commit/PR to provide it for testing.

Regarding 1.1.0rc3: This release has been made without verifying that the build works. Thus enabling it for every commit would avoid such unexpected issues in the future.

**Comment by FriedrichFroebel** [comment]
I have just released version 1.1.0 which includes both an AppImage and a Flatpak binary after some (unexpected) hassle with the build not working which I have been able to solve before the actual release.
